pkg/webservice/handlers.go

```diff
@@ -898,6 +898,56 @@ func getPartitionRules(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func getQueueApplicationsByState(w http.ResponseWriter, r *http.Request) {
+	writeHeaders(w)
+	vars := httprouter.ParamsFromContext(r.Context())
+	if vars == nil {
+		buildJSONErrorResponse(w, MissingParamsName, http.StatusBadRequest)
+		return
+	}
+
+	partition := vars.ByName("partition")
+	queueName := vars.ByName("queue")
```

Comment:
Could you please handle the unescaped queue name? see following link for example

https://github.com/apache/yunikorn-core/blob/master/pkg/webservice/handlers.go#L679

Reply:
Done. I added the validation logic starting from line 924.